protoc-gen-grpc-gateway/internal/gengateway/template.go

```diff
@@ -237,6 +238,9 @@ var _ status.Status
 var _ = runtime.String
 var _ = utilities.NewDoubleArray
 var _ = metadata.Join
+var _ = github_com_prysmaticlabs_eth2_types.Epoch(0)
+var _ = emptypb.Empty{}
+var _ = empty.Empty{}
```

Comment:
I added these to prevent errors on compilation. I'm not sure why but sometimes the compiled output imports 1 name of emptypb, but uses another different (unimported) name.